README.md

```diff
@@ -11,22 +11,22 @@ reactors.
 
 Moltres documentation can be found at
 http://arfc.github.io/software/moltres. Doxygen pages are
-[here](https://arfc.github.io/moltres/). Outlines of the kernels and boundary
+[here](https://arfc.github.io/moltres/classes.html). Outlines of the kernels and boundary
 conditions used to construct the Moltres governing equations can be found on the
 [Moltres wiki](http://arfc.github.io/software/moltres/wiki/). Breakdown of a
 full-fledged Moltres input file can be found
 [here](http://arfc.github.io/software/moltres/wiki/input_example/). New Moltres
-users who have never used MOOSE before are encouraged to check-out its
-[wiki](http://mooseframework.org/wiki/),
-[tutorials](http://mooseframework.org/wiki/MooseTutorials/), and
-[examples](http://mooseframework.org/wiki/MooseExamples/) to help understand the
+users who have never used MOOSE before are encouraged to check-out the MOOSE
+workshop [slides](https://mooseframework.inl.gov/workshop/index.html#/) and 
+[video](https://www.youtube.com/watch?v=2tJwBsYaLaI) to help understand the
```

Comment:
The link works, but is an 11 hour video the best link to put?

Reply:
Hmm. It's a tough call because MOOSE requires some time investment to gain basic proficiency. Should I just leave a link to the MOOSE website?

Reply:
Yeah. I think that plus the slides would be helpful. I am assuming that the website will have the video linked if a user wants to invest the time.

Reply:
Yea I got the video link from their website. I'll link their website and workshop slides.